### DISCUSSION TITLE
How to distinguish between server and client rendering?

### DISCUSSION THREAD

**Discussion by wizzard0** [flareact/flareact#114] [Q&A] [answered]
I have a component that uses IndexedDB and so can't work during SSR\ESR.

Can I detect the context in flareact and skip rendering the component?

**Comment by traffisco** [comment, marked as answer]
useEffect to the rescue

**Reply by Alkalurops** [reply]
That's a question I had, too – and yes, that works very well! 
It can even be used to conditionally / on demand `import()` a WebGL renderer and initiate a `requestAnimationFrame`-render loop for it 👍
